core/systems/project/node_page.ex

```diff
@@ -40,150 +38,27 @@ defmodule Systems.Project.NodePage do
     }
   end
 
-  # Events
-
-  @impl true
-  def handle_event("rename", %{"item" => item_id}, socket) do
-    item = Project.Public.get_item!(String.to_integer(item_id))
-
-    {
-      :noreply,
-      socket
-      |> assign(focussed_item: item)
-      |> compose_child(:project_item_form)
-      |> show_modal(:project_item_form, :dialog)
-    }
-  end
-
-  @impl true
-  def handle_event("delete", %{"item" => item_id}, socket) do
-    Project.Public.delete_item(String.to_integer(item_id))
-
-    {
-      :noreply,
-      socket
-      |> update_view_model()
-      |> update_menus()
-    }
-  end
-
-  @impl true
-  def handle_event("create_item", _params, socket) do
-    {
-      :noreply,
-      socket
-      |> compose_child(:create_item_view)
-      |> show_modal(:create_item_view, :dialog)
+  def compose(:grid_view, %{} = assigns) do
+    %{
+      module: Systems.Project.NodePageGridView,
+      params: assigns
```

Comment:
Consider to be specific about which assigns are actually being utilized in the child component. Instead of passing all assigns, explicitly pass only the necessary ones to improve clarity, maintainability, and reduce unintended coupling between components.

Reply:
Deze hele method lijkt niet meer gebruikt te worden, is een overblijfsel van de oude situatie dezelfde logica zit nu in node_page_builder.ex